docs/overrides/main.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% block extrahead %}
+{{ super() }}
+<script>
+  (function(h,o,u,n,d) {
+    h=h[d]=h[d]||{q:[],onReady:function(c){h.q.push(c)}}
+    d=o.createElement(u);d.async=1;d.src=n
+    n=o.getElementsByTagName(u)[0];n.parentNode.insertBefore(d,n)
+  })(window,document,'script','https://www.datadoghq-browser-agent.com/us1/v4/datadog-rum.js','DD_RUM')
+  window.DD_RUM.onReady(function() {
+    window.DD_RUM.init({
+      clientToken: 'pub77d1be21718df74d03f852860d7f6b7f',
```

Comment:
Non-blocking nit: This looks like some template library. Is it possible to use env variable for these 2 configurations at build time?

Reply:
Should be possible to read in environment variables. Seeing as this token and app ID are public information (they will be visible to anybody loading the site), is there benefit to doing that?

Reply:
Feels cleaner to separate data from checked-in codes. Potentially we can also update api key etc. without update the sdk codes if data is kept out. Small things like this. Although if we follow that path, might need to check how to make local instance render fine without it.

Non-blocking and up to your judgement.

Reply:
Went ahead and implemented it. These variables are defined as GitHub repo variables (not secrets), loaded into `mkdocs.yml`, and referenced in this template via `{{ config.extra.dd_rum.client_token }}`. Looks like it works.